### PR TITLE
Identify unknown healthbox GFX

### PR DESCRIPTION
**Pull request by froggestspirit**
Two tiles in the unknown GFX folder were identified, moved and renamed. The files have also been converted from grayscale to their proper indexed palette

### CODE REVIEW

File: src/graphics.c [hunk 1]
```diff
@@ -374,8 +374,8 @@ const u8 gUnknown_08C1249C[] = INCBIN_U8("graphics/battle_interface/ball_display
 const u8 gBattleInterfaceGfx_Status2[] = INCBIN_U8("graphics/battle_interface/status2.4bpp"); // these three duplicate sets of graphics are for the opponent pokemon
 const u8 gBattleInterfaceGfx_Status3[] = INCBIN_U8("graphics/battle_interface/status3.4bpp"); // and are also for use in double battles. they use dynamic palettes so
 const u8 gBattleInterfaceGfx_Status4[] = INCBIN_U8("graphics/battle_interface/status4.4bpp"); // coloring them is an extreme headache and wont be done for now
-const u8 gUnknown_D12FEC[] = INCBIN_U8("graphics/unknown/unknown_D12FEC.4bpp");
-const u8 gUnknown_D1300C[] = INCBIN_U8("graphics/unknown/unknown_D1300C.4bpp");
+const u8 gBattleInterfaceGfx_frameend[] = INCBIN_U8("graphics/battle_interface/healthbox_doubles_frameend.4bpp");
+const u8 gBattleInterfaceGfx_frameend_bar[] = INCBIN_U8("graphics/battle_interface/healthbox_doubles_frameend_bar.4bpp");
```
Comment (GriffinRichards) [inline src/graphics.c:378]: Case: `gBattleInterfaceGfx_FrameEnd`, `gBattleInterfaceGfx_FrameEnd_Bar`

File: src/battle_interface.c [hunk 2]
```diff
@@ -153,8 +153,8 @@ enum
     HEALTHBOX_GFX_STATUS_BRN_BATTLER3, //status4 "BRN"
     HEALTHBOX_GFX_114,
     HEALTHBOX_GFX_115,
-    HEALTHBOX_GFX_116, //unknown_D12FEC
-    HEALTHBOX_GFX_117, //unknown_D1300C
+    HEALTHBOX_GFX_116, //gBattleInterfaceGfx_frameend
+    HEALTHBOX_GFX_117, //gBattleInterfaceGfx_frameend_bar
```
Comment (GriffinRichards) [inline src/battle_interface.c:157]: You can drop the comment here and rename the constant instead, e.g. `HEALTHBOX_GFX_FRAME_END`

**Comment by froggestspirit** [comment]
I think I updated this right, still not too familiar with ammending commits/pull requests

**Comment by GriffinRichards** [comment]
Yes you have it right, any commits you push to the branch you opened the PR with (your master branch) will end up here.

You still have the other change to make.

**Comment by froggestspirit** [comment]
Should be all set now

**Comment by GriffinRichards** [comment]
Thanks!

**Pull request merged**